docs/1-Applying/when_to_apply.md

```diff
@@ -1,9 +1,23 @@
 ## When to apply
 
-If you're applying for summer internship and want to go to a big or well known company, often you'll need to apply September/October-ish the year before. Yes, this is early. Yes, this is too early. Yes, it's ridiculous. Pretty much, companies figure that if they extend you an offer before everyone else, you'll be more likely to work there. But then everywhere has been doing this and then if you wait until January the intern class will be full.
+### Summer internships
 
-However, there are also a lot of companies that will keep hiring in the new year, and some that only start hiring in the new year.
+If you're applying for summer internship you need to look into applying in September/October-ish the year before, which is the start of the recruiting season. Yes, this is early. Yes, this is too early. Yes, it's ridiculous. The more you wait, the more likely slots will get filled up. That's why we recommend applying when recruiting season start. If you wait until January, the intern class could be full.
```

Comment:
Uh good question. I might have been thinking something like "at least look into applying so you don't miss out or have more time to know what you want and prepare for it" but I'd need to specify that and say that "you don't necessarily need to actually apply that early except for big companies". Or I could just add the big/well-known company clause back in.